src/seedu/addressbook/AddressBook.java

```diff
@@ -210,35 +201,35 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
+        processUserInput();
+    }
+
+    private static void processUserInput() {
```

Comment:
well `processUserInput` gives the impression that this is a one-time process. however, the process is in a loop repeats until the user chooses to exit. 
in light of this behavior, you may want to choose a better suited name